examples/docker_provider/main.tf

```diff
@@ -0,0 +1,23 @@
+provider "docker" {
+  host = "${var.docker_host}"
+}
+
+data "docker_registry_image" "chef_client" {
+  name = "errygg/chef-client"
```

Comment:
Ok, I found a "better" one that isn't my own image: https://hub.docker.com/r/rastasheep/ubuntu-sshd/. Will post the changes soon. Doing some final testing.